src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1483,4 +1484,47 @@ Please specify one of them using the `--source` option to proceed.</value>
     <value>Cannot disable %1. This setting is controlled by policy. For more information contact your system administrator.</value>
     <comment>{Locked="%1"} The value will be replaced with the feature name</comment>
   </data>
+  <data name="PinAddCommandLongDescription" xml:space="preserve">
+    <value>Add a new pin. A pin can limit the Windows Package Manager from updating a package to specific ranges of versions, or it can prevent it from updating the package altogether. A pinned package may still update on its own and be updated from outside the Windows Package Manager. By default, a pinned package can be updated by mentioning it explicitly in the 'upgrade' command or by adding the '--include-pinned' flag to 'winget upgrade --all'.</value>
+    <comment>{Locked{"'upgrade'"} Locked{"--include-pinned"} Locked{"winget upgrade --all"}</comment>
+  </data>
+  <data name="PinAddCommandShortDescription" xml:space="preserve">
+    <value>Add a new pin</value>
+  </data>
+  <data name="PinCommandLongDescription" xml:space="preserve">
+    <value>Manage package pins with the sub-commands. A pin can limit the Windows Package Manager from updating a package to specific ranges of versions, or it can prevent it from updating the package altogether. A pinned package may still update on its own and be updated from outside the Windows Package Manager.</value>
+  </data>
+  <data name="PinCommandShortDescription" xml:space="preserve">
+    <!-- TODO: We need to add "package pin[ning]" to the dictionary of terms for the loc team to know before sending this for localization -->

```

Comment:
Have we already done this for the other terms we use, like "source" or "package"?

Reply:
Looks like not. I added pinning to the internal work item tracking these.